lib/setup.py

```diff
@@ -92,7 +92,7 @@ def run(self):
 
 readme_path = THIS_DIRECTORY / ".." / "README.md"
 if readme_path.exists():
-    long_description = readme_path.read_text()
+    long_description = readme_path.read_text(encoding="utf-8")
```

Comment:
What is the reason for this change? Did you run into a problem with this specified?